src/__tests__/groupMembership.test.ts

```diff
@@ -0,0 +1,139 @@
+import Account from '../account'
+// @ts-ignore no type defs exist for js-sdk
+import Tozny from '@toznysecure/sdk/node'
+import { v4 as uuidv4 } from 'uuid'
+import { cleanupRealms } from './utils'
+
+// default of 5s is sometimes, but not always, enough time. give the tests 10s
+let testTimeout = Number(process.env.TEST_TIMEOUT)
+jest.setTimeout(testTimeout)
+
+const accountFactory = new Account(Tozny, process.env.API_URL)
+let client: any = null
+let realmName: string
+let identity: any = null
+let realmConfig: any = null
+let realm: any = null
+let username: any = null
+let apiURL = process.env.API_URL
+
+beforeAll(async () => {
+  // Create an account to re-use across test cases
+  const seed = uuidv4()
+  const name = `Test Account ${seed}`
+  const email = `test+${seed}@tozny.com`
+  const password = uuidv4()
+  const accountClient: any = await accountFactory.register(
+    name,
+    email,
+    password
+  )
+  const sovereignName = 'JELP'
+  realmName = `TestRealm${seed.split('-')[0]}`.toLowerCase()
+
+  const tokenName = 'example token'
+  const permissions = {
+    enabled: true,
+    allowed_types: ['general', 'identity'],
+  }
+  client = accountClient.client
+  const token = await client.newRegistrationToken(tokenName, permissions)
+  const stringToken = String(token.token)
+  const createdRealm = await client.createRealm(realmName, sovereignName)
+  username = `it-user-${uuidv4()}`
+  realmConfig = {
+    realmName: realmName,
+    appName: 'account',
+    brokerTargetUrl: 'http://integrationtest.local.tozny.com',
+    apiURL,
+  }
```

Comment:
this seems unused in the rest of the test file. i'd remove it, or at least add a `const` to make it block scoped to this beforeAll hook